iodata/iodata.py

```diff
@@ -30,36 +31,9 @@
 __all__ = ['IOData']
 
 
-def convert_array_to(dtype):
```

Comment:
I like that these two functions are not part of iodata.py anymore and are now in atrs utils. This is cleaner and easier to follow.

Reply:
Yep, they were indeed not in the right place, a bit too intimidating when one just wanted to see how the IOData class works. :)